salt/core/init.sls

```diff
@@ -23,3 +23,16 @@ root_authorized_keys_add:
   ssh_auth.present:
     - user: root
     - source: salt://private/authorized_keys/root_to_add
+
+include:
+    - core.apt
+    - core.customization
+    - core.fail2ban
+    - core.firewall
+    - core.locale
+    - core.mail
+    - core.motd
+    - core.ntp
+    - core.sshd
+    - core.swap
+    - core.systemd
```

Comment:
These aren't dependencies of the `init.sls` file. I think it's better to leave them in the top file.

Reply:
Very true, I was considering them dependencies of the "core" install. I don't see us running `init.sls` without the other files.
The benefit of moving these here is that when `onboarding.sls` runs `include: core` this then picks up on all of the other state files. If we leave this in top we'll need to duplicate the list of state files in `onboarding.sls`.

Reply:
I see you're other comment regarding `salt-ssh TARGET 'onboarding,core*'` which also mitigates this. 
Happy with that approach instead.

Reply:
Very true, I was considering them dependencies of the "core" install as I don't see us running `init.sls` without the other files.

The benefit of moving these here is that when `onboarding.sls` runs `include: core` this then picks up on all of the other state files. Rather than duplicating them in `onboarding.sls`.
I see your other comment regarding `salt-ssh TARGET 'onboarding,core*'` which also mitigates this.
Happy with that approach instead.